receiver/azureeventhubreceiver/README.md

```diff
@@ -50,6 +50,18 @@ attribute names are copied without any changes.
 
 Default: `false` (semantic conventions are not applied)
 
+### time_format (optional)
+
+All supported time format for logs, metrics and traces. Default is empty string array, which means using the current iso8601 parser. The format is based on https://pkg.go.dev/time#Layout. If no time-zone info, will use UTC time.
+
+Default: `nil`
+
+### time_offset (optional)
+
+The offset hours to parsed time. Mainly for cases when there's no time-zone info in time string.
```

Comment:
offset to what? UTC?

Reply:
this seems like something a processor needs to do?

Reply:
If the time string doesn't contains time-zone info, it'll be parsed as UTC time. Offset will do offset directly no matter it's UTC time or others. It aims to solve the case that time string without time-zone info and being parsed to wrong time.
Per your suggestion, I removed `time_offset` logics and will see if processor could handle. If not, I'll submit another PR for it.